seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/util/CalculateUtils.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.seonoh.seonohapp.util
+
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import com.example.seonoh.seonohapp.R
+import java.text.DecimalFormat
+
+//onemask 코드 사용 ( 자체 리뷰 )
+
+val doubleFormat = DecimalFormat("#,##0.00000000")
+val intFormat = DecimalFormat("#,###.##")
+
+//마켓 이름
+fun TextView.setMarketName(marketName: String?) {
+    if(marketName!!.contains("USDT")){
```

Comment:
```suggestion
text = marketName?.substringAfter("-") ?: ""
```

- 파라미터가 nullable이면 !! 보단 `?.` , `?:` 연산자를 사용
- length 별로 substring의 카운트를 달리 하는 것보다는 subStringAfter(delimiter) 함수가 가독성이 좋아보입니다.

Reply:
네 확인했습니다.